index.js

```diff
@@ -51,6 +51,8 @@ const main = async (options = {}) => {
 			pid: Number.parseInt(key, 10),
 			name: path.basename(value.comm),
 			cmd: value.args,
+			path: value.comm,
+			args: value.args.split(value.comm)[1].trim(),
```

Comment:
I would prefer using `startsWith` and `slice` here. The current way will fail if the `args` contain an argument that is the same as `comm`.

Reply:
I don't get you. 1) `comm` only returns the path name; 2) if they are the same, that means there is no argument and it returns empty string